frontend/packages/dev-console/src/components/import/ImportForm.tsx

```diff
@@ -68,6 +69,9 @@ const ImportForm: React.FC<ImportFormProps & StateProps> = ({
         privateKey: '',
       },
     },
+    serverless: {
+      allowZeroPods: false,
```

Comment:
I checked Joshua's PR and he has used trigger. I used the same here.

Reply:
ah got it, in mine PR have used `enabled` so will follow one. Whichever goes in first.

Reply:
I think `enabled` makes more sense because we have `build.triggers` and `deployment.triggers` which are used as actual triggers for build or deployment configs.

Reply:
deploy image PR has already gone into master with `trigger`. @christianvogt, @joshuawilson what do you think?

Reply:
It's outdated, you have already changed this. That means that you can mark it resolved.

Reply:
@joshuawilson initially i have defined as `allowZeroPods`. After referring your PR i used `trigger`. Now your PR is already merged with `trigger`. Now there is a suggestion to use `enabled`. Should I change all the reference to `enabled` from `trigger`?

Reply:
That can be changed in a refactoring of the code.

Reply:
ok